From 9f4c0b23efc4571eefa6ba1a8abf1088b098b84b Mon Sep 17 00:00:00 2001
From: Wen Shi <wen.shi@windriver.com>
Date: Thu, 1 Feb 2018 16:45:15 -0600
Subject: [PATCH 78/91] Update ceilometer config to support tox coverage

---
 tox.ini | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tox.ini b/tox.ini
index 10a211e..09a0977 100644
--- a/tox.ini
+++ b/tox.ini
@@ -45,9 +45,10 @@ passenv = CEILOMETER_*
 commands = pifpaf run mongodb {toxinidir}/tools/pretty_tox.sh "{posargs}"
 
 [testenv:cover]
-setenv = OS_TEST_PATH=ceilometer/tests
 commands =
+   coverage erase
    python setup.py testr --slowest --coverage --testr-args="{posargs}"
+   coverage xml
 
 [testenv:pep8]
 deps = hacking<0.13,>=0.12
-- 
2.7.4

